stories/ui/RecommendedBadge.js

```diff
@@ -0,0 +1,15 @@
+/* @flow */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import RecommendedBadge from 'ui/components/RecommendedBadge';
+
+import Provider from '../setup/Provider';
+
+storiesOf('RecommendedBadge', module)
+  .addDecorator((story) => (
```

Comment:
Is this decorator needed?

Reply:
I was using it to provide an instance of `i18n`, but I guess I can just do that directly. I have updated the patch to do that.